### DISCUSSION TITLE
Patterns to use for removing deleted filed

### DISCUSSION THREAD

**Discussion by tobiascarlen** [newren/git-filter-repo#556] [Q&A] [answered]
**git filter-repo --invert-paths --path directory/name.json** works fine, but **git filter-repo --invert-paths --path directory/*.json** and **git filter-repo --invert-paths --path directory/.json** don't...   

What's the right way to use wildcards here, or do i need to pipe in the file names instead?,

**Comment by newren** [comment, marked as answer]
If you want to use globs, use `--path-glob`.  If you want to use regexes, use `--path-regex`.  `--path` looks for an exact match, so `--path directory/*.json` is only going to find a file literally named `directory/*.json` and no other files under `directory/` with a `.json` ending.
